app/views/layouts/_nav.html.erb

```diff
@@ -18,6 +19,20 @@
   <div class="collapse navbar-collapse" id="scp-navbar-dropdown-collapse">
     <%= render partial: '/layouts/breadcrumbs' %>
     <ul class="nav navbar-nav pull-right">
+      <li>
+        <%= scp_link_to "<span class='fas fa-plus fa-fw'></span> Create a Study".html_safe, new_study_path, class: 'check-upload', 'data-analytics-name' => 'prominent-create-study-button' %>
+      </li>
+      <% if BrandingGroup.visible_groups_to_user(current_user).any? %>
+        <li class='dropdown'><%= scp_link_to "Collections <span class='caret'></span>".html_safe, "#", class: "dropdown-toggle", "data-toggle" => "dropdown" %>
+          <ul class="dropdown-menu dropdown-menu-left">
+            <li class="left-border-0-5"><%= link_to '<span class="fas fa-globe"></span> Directory'.html_safe, collections_path %></li>
```

Comment:
Just me, but I'm not crazy about the globe as the icon here.  It doesn't really convey a meaning to me that seems to indicate what this page is going to be.  Would be curious to see other options.

Reply:
concur -- maybe some sort of list icon?